lib/src/oauth2.dart

```diff
@@ -207,7 +207,7 @@ Future<Client> _authorize() async {
     completer
         .complete(grant.handleAuthorizationResponse(queryToMap(queryString)));
 
-    return shelf.Response.found('https://pub.dartlang.org/authorized');
+    return shelf.Response.found('https://pub.dev/authorized');
```

Comment:
Needs to be confirmed that it is OK to change this

Reply:
I think this okay :D